pkg/applicationserver/applicationserver.go

```diff
@@ -1434,7 +1397,8 @@ func (as *ApplicationServer) GetMQTTConfig(ctx context.Context) (*config.MQTT, e
 
 // RangeUplinks ranges the application uplinks and calls the callback function, until false is returned.
 func (as *ApplicationServer) RangeUplinks(ctx context.Context, ids *ttnpb.EndDeviceIdentifiers, paths []string, f func(ctx context.Context, up *ttnpb.ApplicationUplink) bool) error {
-	return as.appUpsRegistry.Range(ctx, ids, paths, f)
+	// TODO: Implement this.
```

Comment:
This can be dropped - no other upstreams outside of the GLS integration depend on it.